src/app/cli/tests/tests/test_repository_alias_commands.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use opendatafabric::*;
+
+use crate::utils::Kamu;
+
+#[test_log::test(tokio::test)]
+async fn test_repository_alias_commands() {
```

Comment:
Naming proposition: `test_repository_remove_all_pull_aliases()`

Reply:
Will add here push alias and will keep this naming